pyfar/constants/speed_of_sound.py

```diff
@@ -0,0 +1,41 @@
+"""File containing all speed of sound calculation functions."""
+import numpy as np
+
+
+def speed_of_sound_simple(temperature):
+    r"""
+    Calculate the speed of sound in air using a simplified version
+    of the ideal gas law based on the temperature.
+
+    Calculation is in accordance with ISO 9613-1 [#]_, as described in
+    ISO 17497-1 [#]_.
```

Comment:
Why is the second reference necessary?
```suggestion
    Calculation is in accordance with ISO 9613-1 [#]_.
```